default/stringtables/es.txt

```diff
@@ -2738,7 +2738,7 @@ OPTIONS_FONT_BOLD_TEXT
 Texto en negrita
 
 OPTIONS_FONT_TITLE
-Títulos de ventanas
+Ventanas
```

Comment:
I think that if "Títulos de ventana" (Windows titles) doesn't fit in the standard window size, you could use instead "Títulos" (Titles) rather than "Ventanas" (Windows). What do you think @geoffthemedio @Vezzra ?

Reply:
"Titles" seems more helpful than "Windows" in this context

Reply:
Agree with @geoffthemedio...

Reply:
I ended up undoing the change, it's now Window Titles (Tñitulos de Ventana)

Reply:
Ok!